client/src/components/overview/image/PickerImage.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+function PickerImage({
+  idx,
+  currID,
+  url,
+  setImage,
+  transform,
+}) {
+  const [src, setSrc] = React.useState(url);
+
+  React.useEffect(() => {
+    setSrc(url);
+  }, [url]);
+
+  const onError = () => {
+    setSrc('https://www.freeiconspng.com/uploads/no-image-icon-6.png');
+  };
+
+  return (
+    <span
+      className="pickerImageBG"
+      style={{
+        objectFit: 'cover',
+        transform: `translate3d(0, ${-transform * 10}vh, 0)`,
```

Comment:
It's a bit confusing that you have a variable named transform that is used with transform property, maybe it can have a more descriptive name to make it clear what it represents.